src/rust/engine/src/context.rs

```diff
@@ -38,6 +40,16 @@ impl Core {
     let mut snapshots_dir = PathBuf::from(work_dir);
     snapshots_dir.push("snapshots");
 
+    let store_path = work_dir.join("lmdb_store");
+    let store = std::fs::create_dir_all(&store_path)
```

Comment:
To allow for concurrent creation (we'll want to locate this in the homedir soon), see https://github.com/pantsbuild/pants/blob/4d476f75d8ad58ef826908aff81e8cc1fd71e821/src/rust/engine/fs/src/lib.rs#L836

Reply:
Done

Reply:
Also moved to ${HOME}/.cache/pants/lmdb_store - will hold off on merging until you OK this path